xarray/coding/cftime_offsets.py

```diff
@@ -849,14 +855,72 @@ def _generate_range(start, end, periods, offset):
             current = next_date
 
 
+class _NoDefault(Enum):
```

Comment:
Should we move this to `pdcompat.py` so that the copyright notice is attached?

Reply:
We have the pandas copyright notice in `cftime_offsets.py` as well, but I agree this seems like something more general and not necessarily immediately referred to by [this note](https://github.com/pydata/xarray/blob/f46cd708f8e220272173e2fc3e66c7688df45c39/xarray/coding/cftime_offsets.py#L2-L5), so better to be in `pdcompat.py`.